### PR TITLE
WEBDEV-5694 Make mediatype icons in list view link to their collection

### PR DESCRIPTION
**Pull request by latonv**
Converts extended list view mediatype icons into links targeting their mediatype's collection details page.

### CODE REVIEW

File: src/tiles/list/tile-list.ts [hunk 1]
```diff
@@ -331,6 +331,23 @@ export class TileList extends LitElement {
...
   }
 
+  /** The URL of this item's mediatype collection, if defined. */
+  private get mediatypeURL(): string | typeof nothing {
+    if (!this.baseNavigationUrl || !this.model?.mediatype) return nothing;
+
+    // Need special handling for certain mediatypes that don't have a top-level collection page
+    switch (this.model.mediatype) {
+      case 'collection':
+        return `${this.baseNavigationUrl}/search?query=mediatype:collection&sort=-downloads`;
```
Comment (dualcnhq) [inline src/tiles/list/tile-list.ts:341]: TIL `sort=-downloads`

File: src/tiles/list/tile-list.ts [hunk 2]
```diff
@@ -120,13 +120,13 @@ export class TileList extends LitElement {
   // Data templates
   private get iconRightTemplate() {
     return html`
-      <div id="icon-right">
+      <a id="icon-right" href=${this.mediatypeURL} target="_blank">
```
Comment (dualcnhq) [inline src/tiles/list/tile-list.ts:123]: :nice:

File: src/tiles/list/tile-list.ts [hunk 3]
```diff
@@ -331,6 +331,23 @@ export class TileList extends LitElement {
...
+    if (!this.baseNavigationUrl || !this.model?.mediatype) return nothing;
+
+    // Need special handling for certain mediatypes that don't have a top-level collection page
+    switch (this.model.mediatype) {
+      case 'collection':
+        return `${this.baseNavigationUrl}/search?query=mediatype:collection&sort=-downloads`;
+      case 'account':
+        return nothing;
+      default:
+        return `${this.baseNavigationUrl}/details/${encodeURI(
```
Comment (dualcnhq) [inline src/tiles/list/tile-list.ts:345]: :nice:

**Review by dualcnhq — APPROVED** [review]

File: src/tiles/list/tile-list.ts (hunk 1 above)
Comment (latonv) [reply src/tiles/list/tile-list.ts:341]: I think the home page uses this strategy too for the collection icon in the hero block (list most viewed collections). 🙂

**Pull request merged**